### ISSUE TITLE
2.5.8: Further Examples Required

### ISSUE THREAD

**Issue by benja11y** [w3c/wcag#1872]
To fully understand the impact this SC will have, further examples should be provided that include complex widgets that contain multiple, but separate, discrete targets.  Examples could include [material design style navigation drawers](https://material.io/components/navigation-drawer), date pickers, compact tab controls, ARIA `<select>`-style controls or interactive toast messages that partially obscure content underneath. 

This issue has been raised on behalf of [TPGi](https://www.tpgi.com), based on an internal consultation process. [An article containing all of our comments](https://www.tpgi.com/wcag-22-working-draft-feedback/) can be found on our blog, along with an introduction and acknowledgments.

**Comment by detlevhfischer** [comment]
@benja11y I had a look at the various material design components and nothing strikes me as potenial fail here. Elements may be smaller than 24 x 24 px but there seems to be always ample spacing. So I am just curious where you see the benefit of adding examples from that design vocabulary. I would imagine edge cases ould be more important to cover.

**Comment by detlevhfischer** [comment]
I just checked a few of the ARIA Authoring practices examples:
[Date picker aspin button example:](https://www.w3.org/TR/wai-aria-practices-1.1/examples/spinbutton/datepicker-spinbuttons.html)
Spin buttons 20 x 20 px but sufficient space above and plenty sidewards

Combobox examples:
The example below shows line height is set to 24 px so it just meets the SC:
![combobox-listbox](https://user-images.githubusercontent.com/9989439/121702715-69afbd80-cad2-11eb-877f-e29d5dcadbae.gif)

The same seems to be the case on the other combobox examples. If there are any examples you'd like to call out, let me know. So far I do not see what adding further examples from either source will add.

**Comment by becka11y** [comment]
@benja11y I believe the above comment is for you :-)

**Comment by alastc** [comment]
Closing with #2024
